src/device/pf_cmrpc.c

```diff
@@ -787,7 +787,7 @@ void pf_set_error (
 {
    LOG_INFO (
       PNET_LOG,
-      "CMRPC(%d) ERROR: %u, %u, %u, %u\n",
+      "CMRPC(%d) Setting error: 0x%02X 0x%02X 0x%02X 0x%02X\n",
```

Comment:
Maybe it is a good idea to also log if another error is already set and the currently set error code?

Reply:
Done